CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -46,6 +46,9 @@ local vr3dGuis = (vr3dGuisSuccess and vr3dGuisFlagValue == true)
 local getNewNotificationPathSuccess, newNotificationPathValue = pcall(function() return settings():GetFFlag("UseNewNotificationPathLua") end)
 local newNotificationPath = getNewNotificationPathSuccess and newNotificationPathValue
 
+local newChatVisiblePropSuccess, newChatVisiblePropValue =  pcall(function() return settings():GetFFlag("ChatVisiblePropertyEnabled") end)
+local newChatVisibleProp = (newChatVisiblePropSuccess and newChatVisiblePropValue)
```

Comment:
this seems fine to control with a flag, but lets make sure we don't turn the flag on until this property exists on all platforms on prod